.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -8,34 +8,40 @@ body:
-        Failure to adhere to these guidelines will result in your issue being closed without response.
+        ## Reminder
+        To fix your problem as soon as possible, please go ahead and describe your problem as accurately and concisely.
   
-        **Feature Requests:** Do not submit them here. Use [Discussions](https://github.com/Floorp-Projects/Floorp/discussions/new?category=feature-request).
+        **Feature Requests:** Use [Discussions](https://github.com/Floorp-Projects/Floorp/discussions/new?category=feature-request) instead of GitHub Issues.
         
         For Japanese: 日本人の場合は、日本語で書いても構いません。
   
         ## Disclaimer
-        **Please note:**
-        - No support for operating systems below Windows 10.
-        - DRM content is not supported. Do not report DRM-related issues.
+        > [!NOTE]
+        > Windows 7, 8, 8.1, and earlier versions are not supported due to maintenance costs.
+        > DRM is not supported esp. on Netflix.
+        > In addition, kindly refer to the [Issue Tracker](https://docs.floorp.app/en/issues/)
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Before submitting your issue
+        Kindly refer to the guideline, [When you submit issue to Floorp!](https://docs.floorp.app/en/others/issue-checklist/)
 
   - type: checkboxes
     attributes:
-      label: Pre-Submission Checklist
-      description: Please go through the checklist below before submitting your issue.
+      label: Checklist
+      description: Please go through this before you submit this issue.
       options:
-        - label: I have checked the existing/closed issues to ensure this issue has not already been reported.
-        - label: I am using the latest supported version of the browser.
-        - label: I have tried disabling all browser extensions/add-ons to rule out interference from these.
+        - label: This issue is not duplicated issue in GitHub Issues, Issue Trackers in Floorp Docs.
+        - label: I am using the latest Floorp.
+        - label: I have tried without all browser extensions/add-ons.
         - label: I have tried clearing the browser cache and cookies.
-        - label: I have checked with a new profile and the issue still occurs.
+        - label: I have tried with a new profile.
+        - label: This issue is not occurring in Firefox ESR, Floorp's base.
+        - label: This issue is in only Floorp, not other browsers.
         - label: I have provided detailed step-by-step instructions on how to reproduce the issue.
-        - label: I have included relevant screenshots or console outputs.
-        - label: I have checked this problem is not occurring in the latest version of Firefox ESR.
-        - label: This issue is specific to this browser and does not occur in other browsers.
-
+        - label: I have provided relevant screenshots or console outputs.
+        
```

Comment:
Please restore the previous checklist. There were no significant changes in this PR to it,